# Node rate config values
detector: 10.0 # rate to run the cup detector note (hz)
game: 10.0 # rate to run the game node at (hz)
tf_broadcast: 20.0 # rate to run the game node at (hz)
socket: 50.0 # rate to run the socket telemetry node at (hz)
calibrator: 1.0 # rate to run the calibrator node at (hz)
